app/index.js

```diff
@@ -81,6 +81,7 @@ module.exports = class GeneratorLicense extends Generator {
         name: 'license',
         message: 'Which license do you want to use?',
         default: this.options.defaultLicense,
+        when: this.options.license == null,
```

Comment:
Doesn't work, as I don't have the object but the value.
I've now have this, which works in all three test cases for me:
`when: this.options.license == null || licenses.find(x => x.value === this.options.license) === undefined,`

Testcases:
- Without license specified - Prompt
- Not known license specified -> Prompt
- License specified in options -> No prompt